hw/dv/dpi/usbdpi/usbdpi.h

```diff
@@ -27,7 +27,7 @@ typedef uint32_t svBitVecVal;
 // TODO - until such time as we can perform multiple control transfers in a
 //        single bus frame, this is impractical because the simulation takes too
 //        long and is apt to time out
```

Comment:
```suggestion
// TODO - Because we cannot perform multiple control transfers in a single bus
//        frame yet, simulation can take a long time and risk timing out. Users may
//        want to set this to 0 to reduce simulation time. Note that reducing the
//        frame interval is not spec-compliant.
```

Reply:
updated